internal/mic/mic.go

```diff
@@ -61,3 +64,35 @@ func (mici *micImpl) ApplyMIC(ctx context.Context, name, ns string, images []kmm
 
 	return nil
 }
+
+func (mici *micImpl) GetModuleImageSpec(micObj *kmmv1beta1.ModuleImagesConfig, image string) *kmmv1beta1.ModuleImageSpec {
+	for i, imageSpec := range micObj.Spec.Images {
+		if imageSpec.Image == image {
+			return &micObj.Spec.Images[i]
```

Comment:
Do we want to return the actual `ModuleImageSpec` (to be updated outside of this package) or just a copy of it?
If it is the later I would go with something like:
```
for _, imageSpec := range micObj.Spec.Images {
		if imageSpec.Image == image {
			return &imageSpec
                }
}
```

Reply:
Actually, since we only use imageSpec for field inspection, i am switching to your suggestion